recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
I guess that it aggregates all the things we will be editing, when updating the recipe, at the top of the file.